Cargo.lock

```diff
@@ -4,9 +4,9 @@ version = 3
 
 [[package]]
 name = "ahash"
-version = "0.7.4"
+version = "0.7.5"
```

Comment:
These update must have been accidental. Can you undo that please?

`git checkout master -- Cargo.lock` should do the trick.